flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/BuiltInAggregateFunctionTestBase.java

```diff
@@ -204,25 +205,39 @@ TestSpec testResult(
             return this;
         }
 
-        private Executable createTestItemExecutable(TestItem testItem) {
+        private Executable createTestItemExecutable(
+                TestItem testItem, boolean stateBackendChangelogEnabled) {
             return () -> {
+                Configuration conf = new Configuration();
+                conf.set(
+                        StateChangelogOptions.ENABLE_STATE_CHANGE_LOG,
+                        stateBackendChangelogEnabled);
```

Comment:
The changelog is enabled randomly from `flink/pom.xml` unless explicitly disabled (currently, it **is** disabled for this test). So there is no need to enable it explicitly.

Instead, it probably makes sense to parameterize the test with the type of backend (currently it uses HEAP, but would fail with ROCKSDB), ideally.

Reply:
Thanks!